mongo/collection.go

```diff
@@ -595,9 +664,16 @@ func (coll *Collection) ReplaceOne(ctx context.Context, filter interface{},
 	return coll.updateOrReplace(ctx, f, r, false, rrOne, false, updateOptions...)
 }
 
-// Aggregate runs an aggregation framework pipeline.
+// Aggregate executes an aggregate operation (https://docs.mongodb.com/manual/reference/command/aggregate/) against
+// the collection.
 //
-// See https://docs.mongodb.com/manual/aggregation/.
+// The pipeline parameter must be an array of documents, each representing an aggregation stage. The pipeline cannot
+// be nil but can be empty. The stage documents must all be non-nil. For a pipeline of bson.D documents, the
+// mongo.Pipeline type can be used. See
+// https://docs.mongodb.com/manual/reference/operator/aggregation-pipeline/#db-collection-aggregate-stages for a list of
+// valid stages in aggregations.
+//
+// The opts parameter can be used to specify options for the operation (see the options.AggregateOptions documentation.)
 func (coll *Collection) Aggregate(ctx context.Context, pipeline interface{},
```

Comment:
Might be obvious, but consider specifying what the method returns as well.

Reply:
done.